app/utils/estimate_prioritization/pooling_functions.py

```diff
@@ -17,6 +18,16 @@ def get_unique_value(series: pd.Series, default: Any = pd.NA) -> Any:
     else:
         return default
 
+def weighted_average(df: pd.DataFrame, values: str, weights: str) -> float:
+    df = df.dropna(axis = 'index', 
```

Comment:
Best practice to not reassign variables that are present in the function signature.

Could call the first arg in the function signature `input_df` instead

Reply:
done